galata/test/jupyterlab/terminal.test.ts

```diff
@@ -84,3 +84,14 @@ test.describe('Terminal', () => {
     });
   });
 });
+
+test('Terminal should open in Launcher cwd', async ({ page, tmpPath }) => {
+  await page.waitForSelector(`.jp-Launcher-cwd > h3:has-text("${tmpPath}")`);
+
+  await page.locator('[role="main"] >> p:has-text("Terminal")').click();
+
+  const terminal = page.locator(TERMINAL_SELECTOR);
+  await terminal.waitFor();
+
```

Comment:
Let's print current directory to check we are at the expected position

```suggestion

    await page.keyboard.type('pwd');
    await page.keyboard.press('Enter');
```